apps/apollo-stories/src/components/ContentItemDuoAction/ContentItemDuoAction.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { Meta, StoryObj } from "@storybook/react";
+import bank from "@material-symbols/svg-700/rounded/account_balance_wallet-fill.svg";
+import { ContentItemDuoAction } from "@axa-fr/design-system-apollo-react/lf";
```

Comment:
ici, tu devrais plutot importer celui d'Apollo : 
```suggestion
import { ContentItemDuoAction } from "@axa-fr/design-system-apollo-react";
```

Reply:
done